tests/test_shipment.py

```diff
@@ -211,10 +211,10 @@ def test_shipment_get_lowest_smartrate(basic_shipment):
 
     # Test lowest smartrate with valid filters
     lowest_smartrate_filters = easypost.Shipment.get_lowest_smartrate(
-        smartrates, delivery_days=1, delivery_accuracy="percentile_90"
+        smartrates, delivery_days=2, delivery_accuracy="percentile_90"
```

Comment:
## Reviewer Note:

After re-recording the cassettes, the smart rate delivery day got changed, update of the unit test is needed.